video_xblock/backends/vimeo.py

```diff
@@ -66,3 +186,88 @@ def player_data_setup(context):
             "vimeo": {"iv_load_policy": 1},
         })
         return result
+
+    def get_default_transcripts(self, **kwargs):
```

Comment:
I tried adding a Vimeo video that shows captions when viewed on Vimeo.  https://vimeo.com/76979871
What is needed to display them in the xblock video player?